bentoml/cli/bento_service.py

```diff
@@ -167,13 +167,27 @@ def open_api_spec(bento=None, yatai_url=None):
         help='Remote YataiService URL. Optional. '
         'Example: "--yatai-url http://localhost:50050"',
     )
+    @click.option(
+        '--enable-swagger/--disable-swagger',
+        is_flag=True,
+        default=True,
+        help="Run API server with Swagger UI enabled",
+        envvar='BENTOML_ENABLE_SWAGGER',
```

Comment:
BentoML uses two underscores between sections of configuration, like `BENTOML__CORE__DEBUG`.

I can see users might make mistakes when they provide this option as envvar and other BentoML configurations as envvars.